lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
```

Comment:
great! I like that you have delegated the task of hosting the menu to a separate class .. follows the principles of object-orientated programming.